src/estimation-kdl/CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ target_link_libraries(${libraryname} idyntree-core idyntree-kdl idyntree-sensors
 get_property(IDYNTREE_TREE_INCLUDE_DIRS GLOBAL PROPERTY IDYNTREE_TREE_INCLUDE_DIRS)
 target_include_directories(${libraryname} BEFORE PRIVATE ${IDYNTREE_TREE_INCLUDE_DIRS})
 
+if((${CMAKE_CXX_COMPILER_ID} MATCHES "GNU") OR (${CMAKE_CXX_COMPILER_ID} MATCHES "Clang"))
+    target_compile_options(${libraryname} PRIVATE "-Wno-deprecated")
```

Comment:
If it was my decision, I would support only current stable (and the debian guys consider stable really stable stuff). But I'm a rolling release user since many years, I'm very biased xD